pcdsdevices/interface.py

```diff
@@ -524,14 +525,23 @@ def __init__(self, *args, **kwargs):
         self._last_status.set_finished()
         super().__init__(*args, **kwargs)
 
+    def _log_move_limit_error(self, position, ex):
+        logger.error('Failed to move %s from %s to %s: %s', self.name,
+                     self.wm(), position, ex)
+
     def _log_move(self, position):
         logger.info('Moving %s from %s to %s', self.name, self.wm(), position)
 
     def _log_move_end(self):
         logger.info('%s reached position %s', self.name, self.wm())
 
-    def move(self, *args, **kwargs):
-        st = super().move(*args, **kwargs)
+    def move(self, position, *args, **kwargs):
```

Comment:
forcing a positional arg named position is breaking tests for the ccm psuedomotor

Reply:
_Bah_, thanks.